Packs/IPQualityScore/Integrations/IPQualityScore/IPQualityScore.py

```diff
@@ -0,0 +1,253 @@
+import urllib.parse
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+''' IMPORTS '''
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' CONSTANTS '''
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+
+class Client(BaseClient):
+
+    def get_ip_reputation(self, ip: str) -> Dict[str, Any]:
+        return self._http_request(
+            method='GET',
+            url_suffix=f'{ip}'
+        )
+
+    def get_email_reputation(self, email: str) -> Dict[str, Any]:
+        return self._http_request(
+            method='GET',
+            url_suffix=f'{email}'
+        )
+
+    def get_url_reputation(self, url: str) -> Dict[str, Any]:
+        return self._http_request(
+            method='GET',
+            url_suffix=f'{url}'
+        )
+    #
+    # def get_url_reputation(self, url: str) -> Dict[str, Any]:
+    #     pass
+
+
+def get_reputation_reliability(reliability):
+    if reliability == "A+ - 3rd party enrichment":
+        return DBotScoreReliability.A_PLUS
+    if reliability == "A - Completely reliable":
+        return DBotScoreReliability.A
+    if reliability == "B - Usually reliable":
+        return DBotScoreReliability.B
+    if reliability == "C - Fairly reliable":
+        return DBotScoreReliability.C
+    if reliability == "D - Not usually reliable":
+        return DBotScoreReliability.D
+    if reliability == "E - Unreliable":
+        return DBotScoreReliability.E
+    if reliability == "F - Reliability cannot be judged":
+        return DBotScoreReliability.F
+
+
+def test_module(client):
+    result = client.get_ip_reputation("8.8.8.8")
+    if result.get('success', False):
+        return 'ok'
+    else:
+        return result.get("message", result)
+
+
+def ip_command(client, args, ip_suspicious_score_threshold, ip_malicious_score_threshold, reliability):
+    ips = argToList(args.get("ip"), ",")
+    results = []
+    for ip in ips:
+        result = client.get_ip_reputation(ip)
+        result['address'] = ip
+
+        human_readable = tableToMarkdown(f"IPQualityScore Results for {ip}", result, result.keys())
+
+        if result.get('fraud_score', 0) >= ip_malicious_score_threshold:
+            score = 3
+            result['Malicious'] = {'Vendor': 'IPQualityScore'}
```

Comment:
```suggestion
```
no need for this line (also in the rest of the commands) - [reason](https://github.com/demisto/content/blob/master/Packs/Base/Scripts/CommonServerPython/CommonServerPython.py#L2407)